crates/transaction-emitter-lib/src/transaction_generator/call_custom_modules.rs

```diff
@@ -58,7 +58,13 @@ impl TransactionGenerator for CallCustomModulesGenerator {
         let mut accounts_to_burn = if let Some(accounts_pool_lock) = &self.accounts_pool {
             let mut accounts_pool = accounts_pool_lock.write();
             let num_in_pool = accounts_pool.len();
-            assert!(num_in_pool > needed, "Left in pool {}, needed {}", num_in_pool, needed);
+            if num_in_pool < needed {
+                sample!(
+                    SampleRate::Duration(Duration::from_secs(120)),
```

Comment:
nit: every 2 minutes seems pretty high, right? Why not be more aggressive, e.g., 10 secs?